src/Sarif/ConsoleLogger.cs

```diff
@@ -186,7 +186,8 @@ private void WriteToConsole(ResultLevel level, Uri uri, Region region, string ru
             if (region != null)
             {
                 // TODO 
-                if (region.Offset > 0 ||
+                if (region.CharOffset > 0 ||
+                    region.ByteOffset > 0 ||
                     region.StartColumn == 0)
```

Comment:
This change is correct in that it is consistent with the existing logic. The existing logic says "If you don't give me a start column, or if you _do_ give me an offset, then I don't know how to format the region string". I don't think you should throw, though. I'd just produce an empty string.

Reply:
Done